src/state/laboratories/actions.ts

```diff
@@ -142,10 +139,6 @@ export const searchLabs =
         examName: query.examName,
         patientCode: !isNaN(query.patientCode) ? query.patientCode : undefined,
       })
-      .pipe(
-        switchMap((labs) => getRowsForPrint(labs)),
-        catchError((error) => of([]))
-      )
```

Comment:
Why do you remove this?

Reply:
The api schema has been updated in order to include Rows In response, so it is not necessary to get the rows again.